tests/datasets/test_eurosat.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Were `AnnualCrop_1.tif` and `Forest_1.tif` intentionally added or should they be in the zip file?

Reply:
They are fake test data and are in the zip file as well

Reply:
But do they need to be outside of the zip file too?

Reply:
I don't think so, but in other datasets we have added the fake data outside of the zip

Reply:
In the more recent classification datasets I didn't keep the data outside the zip.

Reply:
The reason that some datasets have fake data outside the zip is that those datasets have trainers we need to test. Of course we could refactor things to copy and extract the zip files, but that's why things are the way they are.